database/src/tests.rs

```diff
@@ -0,0 +1,94 @@
+//! Utilities for `cuprate_database` testing.
+//!
+//! These types/fn's are only:
+//! - enabled on #[cfg(test)]
+//! - only used internally
```

Comment:
## Tests

- Updated a bunch of tests
- Added top-level `tests.rs` file for testing util functions